pkg/tracking/store/sql/runs.go

```diff
@@ -581,7 +581,19 @@ func ensureRunName(runModel *models.Run) *contract.Error {
 
 func (s TrackingSQLStore) GetRun(ctx context.Context, runID string) (*protos.Run, *contract.Error) {
```

Comment:
I don't think this get the `exactly one` validation they have in Python:

```
        if len(runs) == 0:
            raise MlflowException(f"Run with id={run_uuid} not found", RESOURCE_DOES_NOT_EXIST)
        if len(runs) > 1:
            raise MlflowException(
                f"Expected only 1 run with id={run_uuid}. Found {len(runs)}.",
                INVALID_STATE,
            )
```

Reply:
Works for me, just spotted it when comparing to the Python code.
A weird check indeed when `run_uuid` is probably primary key anyway.